src/site/xdoc/server/config-cassandra.xml

```diff
@@ -156,6 +156,13 @@
             by turning it off. Note that reads performed as part of write transactions are always performed with a strong
             consistency.</dd>
 
+        <dt><strong>message.write.strong.consistency.unsafe</strong></dt>
+        <dd>Optional. Boolean, defaults to true. Disabling should be considered experimental and unsafe.
```

Comment:
Which is this one!

Reply:
I don't understand. Can you develop?

Reply:
cf https://github.com/apache/james-project/pull/350#discussion_r603829405